<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortAssistant</class>
 <widget class="QWidget" name="SerialPortAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;</string>
         </property>
         <property name="text">
          <string>接收窗口</string>
         </property>
         <property name="buddy">
          <cstring>receiveText</cstring>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="receiveNewLine">
         <property name="toolTip">
          <string>每接收一条新数据，将自动换行并在下一行显示内容，而不是挤在一坨</string>
         </property>
         <property name="text">
          <string>新行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="receiveUtf8">
         <property name="toolTip">
          <string>以 UTF-8 编码发接收文本</string>
         </property>
         <property name="text">
          <string>UTF-8</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="receiveGbk">
         <property name="toolTip">
          <string>以 GBK 编码接收文本</string>
         </property>
         <property name="text">
          <string>GBK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="receiveHex">
         <property name="toolTip">
          <string>以 16 进制编码接收数据</string>
         </property>
         <property name="text">
          <string>HEX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearReceive">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="receiveText"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;</string>
         </property>
         <property name="text">
          <string>发送窗口</string>
         </property>
         <property name="buddy">
          <cstring>sendText</cstring>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="sendNewLine">
         <property name="toolTip">
          <string>每发送一个数据，自动在末尾添加回车换行符 (\r\n)（如果没有）</string>
         </property>
         <property name="text">
          <string>新行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sendUtf8">
         <property name="toolTip">
          <string>以 UTF-8 编码发送文本</string>
         </property>
         <property name="text">
          <string>UTF-8</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sendGbk">
         <property name="toolTip">
          <string>以 GBK 编码发送文本</string>
         </property>
         <property name="text">
          <string>GBK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sendHex">
         <property name="toolTip">
          <string>以 16 进制编码发送数据</string>
         </property>
         <property name="text">
          <string>HEX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearSend">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="sendText"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="send">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>按 Ctrl + Enter 键发送</string>
         </property>
         <property name="text">
          <string>发送数据</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>receiveText</tabstop>
  <tabstop>receiveHex</tabstop>
  <tabstop>clearReceive</tabstop>
  <tabstop>sendText</tabstop>
  <tabstop>sendHex</tabstop>
  <tabstop>clearSend</tabstop>
  <tabstop>send</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clearSend</sender>
   <signal>clicked()</signal>
   <receiver>sendText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearReceive</sender>
   <signal>clicked()</signal>
   <receiver>receiveText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
